src/OnboardingSPA/components/Drawer/DrawerPanel/DesignColors.js

```diff
@@ -54,6 +142,14 @@ const DesignColors = () => {
 		}
 	}
 
+	/**
+	 * Converts the user selected value into a suitable valid global styles array value
+	 * param -> selectedColorsLocalTemp -> {base: "#ffffff", contrast: "#404040", ... }
```

Comment:
Can the param and return example be stated where the param is declared, line 150,151? Same concern for other functions too.
and have `@return` also documented in the respective functions?

Reply:
Done